app/controllers/roles_controller.rb

```diff
@@ -0,0 +1,51 @@
+class RolesController < ApplicationController
+
+  def index
+  end
+
+  def new
+    @role = Role.new
+  end
+
+  def create
+    @user_email = role_params[:user_email].downcase
+    @user = User.where(email: @user_email).first
+
+    if @user
+      @course = Course.find(params[:id])
+      @role = Role.new(user: @user, course: @course, role: 1)
```

Comment:
I'm unclear about why the default role is a 1

Reply:
Maybe we should add the role as a parameter in the input form?

Reply:
Yes, we probably need to add the role as an input parameter.

Reply:
Ok, that's fine for now. Maybe we can add an issue to fix this in the future